tools/certify/commands/show-cert.cpp

```diff
@@ -0,0 +1,287 @@
+#include "keyio.hpp"
+#include "show-cert.hpp"
+#include <boost/algorithm/hex.hpp>
+#include <boost/date_time/posix_time/posix_time.hpp>
+#include <boost/program_options.hpp>
+#include <boost/variant.hpp>
```

Comment:
not needed?

Reply:
Removed.

Reply:
Reverted, it's actually needed because of `boost::get()`.